### ISSUE TITLE
[Suggestion] Custom ores and separate stone ores and sand/gravel ores

### ISSUE THREAD

**Issue by Shattered2909** [GregTechCE/GregTech#1102]
It would be really nice to be able to add more blocks (both vanilla and from other mods) for ores to generate in (so I can get things like Umberstone Copper Ore)

Another thing I would like to see is a separation of Stone Ores and non Stone Ores (Gravel, Sand, whatever) so they are/can be processed in different ways, for example Gravel Ore in a Sifter.

**Comment by pyure** [comment]
I'm probably misinterpreting this but...it feels like you're asking to be able to generate non-GT-ores in GT veins, but you can already do this.

**Comment by Shattered2909** [comment]
No, I am asking to be able to generate GT ores for non GT/vanilla stones like Umberstone from the Erebus.

**Comment by Shattered2909** [comment]
The other thing would be separating OreDic between the Ore types so there can be different processing chains for different materials, like sifting Gravel Ores. Probably a good idea to change Oilsands to be a Sand ore by default (atm theres only sandstone ore, would need to change that too)

**Comment by LAGIdiot** [comment]
I probably think where you are going with this, but @Shattered2909 can you please be more specific about this idea there is not much information to go through.

About generating ores. Do you want to let's say define list of blocks in which GTCE could generate it's ores or something similar?

Do you think about separating material based on block they are from (Stone, Granite) or by it's properties (like falling, hardness) of that material (sand/gravel category, rock like category)?

And how would you picture these processing chains? Completely separated or they would join at some point (like after crushing rock and sifting sand  or something similar)?
